### PR TITLE
Fixes tgui tooltip lag, restores the R&D console tooltips

### LINKED ISSUE #60193: The Reformatted Technology Disks from the B.E.P.I.S. create reality-halting gigalag
## Round ID:
[165811](https://scrubby.melonmesa.com/round/165811) on manual and a bunch of others prior

## Reproduction:
1. Get the B.E.P.I.S.
2. Spend money to get diskies
3. Put diskies in R&D console
4. Press load tech from disk to web
5. Experience some lag
6. Press load tech from web to disk
7. Experience painful lag
8. Repeat step 4 to experience +10% time dilation, and a 5 second freeze, more or less
9. Spam step 8

## Why it prally happens:
Disky has some tech on it
loading from server to disk puts even more tech on it
loading an entire RnD server worth of text into the server makes it shit the bed.

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Fixes the tgui tooltip lag by only rendering them when it's being hovered or mid animation.

Closes #60193. (if this 100% fixes tooltip lag)

## Changelog
:cl:
fix: tgui tooltips no longer lag. Restored the R&D console tooltips.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/components/Tooltip.tsx [hunk 1]
```diff
@@ -30,22 +71,29 @@ export class Tooltip extends Component<TooltipProps, TooltipState> {
     // Because this component is written in TypeScript, we will know
     // immediately if this internal variable is removed.
     const domNode = findDOMfromVNode(this.$LI, true);
-
+
```
Comment (stylemistake) [inline tgui/packages/tgui/components/Tooltip.tsx:74]: untrimmed whitespace

Comment (Mothblocks) [reply tgui/packages/tgui/components/Tooltip.tsx:74]:
```suggestion

```

**Comment by Watermelon914** [comment]
sweet

**Review by stylemistake — APPROVED** [review]

**Comment by Mothblocks** [comment]
It's still fairly laggy unfortunately

**Comment by SplinterGP** [comment]
Tooltip unlagging rolecoaster, why are tooltips so laggy ?

**Comment by Mothblocks** [comment]
Every tooltip creates two divs that always exists. This PR removes one, but not the other. Removing the other takes more time but is doable.

**Comment by Mothblocks** [comment]
I need to do this at *some* point for preferences menu, but I don't have the time to get it right away.

**Pull request closed without merging**